src/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -170,11 +171,15 @@ private function getPropertyConstraints(
             }
 
             foreach ($validatorPropertyMetadata->findConstraints($validationGroup) as $propertyConstraint) {
-                $constraints[] = $propertyConstraint;
+                if ($propertyConstraint instanceof Composite) {
+                    $constraints[] = $propertyConstraint->getNestedContraints();
```

Comment:
Wouldn't be better?
```suggestion
                    $constraints = array_merge($constraints, $propertyConstraint->getNestedContraints());
                    continue;
```

And to add back the rest like before?

Reply:
https://github.com/kalessil/phpinspectionsea/blob/master/docs/performance.md#slow-array-function-used-in-loop

Reply:
:+1: never use array_merge in a loop

Reply:
OK!